### PR TITLE
Update requests.rst

### PR DESCRIPTION
**Pull request by christian-hawk**
From issue #239 - Disable SSL verification

### CODE REVIEW

File: docs/client/requests.rst [hunk 1]
```diff
@@ -151,3 +151,33 @@ close the session::
...
+**disable SSL verification**.
+
+You can force all requests to disable SSL verification by setting
+your environment variable ``CURL_CA_BUNDLE=""``.
+
+This solutions works because Python requests (and most of the packages)
+overwrites the default value for ssl verifications from environment
+variables ``CURL_CA_BUNDLE`` and ``REQUESTS_CA_BUNDLE``.
+
+This hack will **only work** with ``CURL_CA_BUDLE``, as you can see
```
Comment (lepture) [inline docs/client/requests.rst:172]: Typo: BUDLE

Comment (christian-hawk) [reply docs/client/requests.rst]: Done @lepture

**Comment by christian-hawk** [comment]
typo fixed

**Comment by lepture** [comment]
Thanks

**Pull request merged**